node/src/chain/acceptor.rs

```diff
@@ -815,6 +831,38 @@ impl<DB: database::DB, VM: vm::VMExecution, N: Network> Acceptor<N, DB, VM> {
             .max(MIN_STEP_TIMEOUT)
             .min(MAX_STEP_TIMEOUT)
     }
+
+    fn emit_metrics(
+        blk: &Block,
+        block_label: &Label,
+        est_elapsed_time: Duration,
+        block_time: u64,
+        block_size_on_disk: usize,
+    ) {
+        let dusk_slashed_count = blk
+            .header()
+            .failed_iterations
+            .to_missed_generators_bytes()
```

Comment:
Can we avoid to produce this if no metrics listen address is set?

Reply:
The same is already done above for printing ` info!("Slashed {}", slashed.to_base58())` . I can add a counter and use it, if you believe that's a bottleneck at all